CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+> These are the canges compared with the legacy fiskaltrust.Launcher.
+
+### Added
+
+- The fiskaltrust.Launcher is now open source.
+- Json launcher configuration file support.
+- Native hosting of GRPC urls.
+- Package download cache.
+- Handling of encrypted configuration values.
+- .NET 6 support.
+- Native executables for Windows, Linux and MacOS.
+- Lunix service installation.
```

Comment:
```suggestion
- Windows service installation.
```

Reply:
I did not include windows service installation because It already exists in the legacy launcher 🤔